modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/resources/com/liferay/dynamic/data/mapping/dependencies/upgrade.v5_3_2/expected-fragment-entry-browser-sniffer-content.html

```diff
@@ -16,4 +16,9 @@
 variableACaseFour = "variableACaseFour"
 variableBCaseFour = "variableACaseFour"
 ]
+
+	<h1> browserSniffer </h1>
```

Comment:
@velosofelipe64, it may not be interpreted correctly by the FreeMarker compiler, can we remove it?